Makefile

```diff
@@ -1,6 +1,9 @@
 BUILD_DIR=$(shell pwd)/build
 CODE_DIR=$(shell pwd)/iso
 VERSION=1.0.0
+GITTAG=$(shell git rev-parse --short HEAD)
+DATE=$(shell date +"%d%m%Y%H%M%S")
+ISO_NAME=minishift-b2d
 
 default: b2d_iso
```

Comment:
If you rop the b2d_iso target, it becomes just _iso_creation_ or _iso_ if you rename the target.